config/schedule.yml

```diff
@@ -78,6 +78,14 @@ award_contributor_badges_from_github:
     - ""
     - award_contributor_from_github
     - ""
+award_community_wellness_badges:
+  description: "Awards 'Community Wellness' badges to users (runs at minute 25 past every 12th hour)"
+  cron: "25 */12 * * *"
+  class: "BadgeAchievements::BadgeAwardWorker"
+  args:
+    - ""
+    - award_community_wellness
+    - ""
```

Comment:
I **think** this job can run maybe twice a day? This was a quick cron setting put together with crontab.guru, so I think there's likely better ways to schedule this.

The queries will check for 1 week windows into the past, so if I make 2 unflagged comments in the "past week", then a week goes by and they fall into the "second week ago" (2.weeks.ago - 1.weeks.ago) we have the "first week ago" (1.weeks.ago - 0.weeks.ago/Time.current) to look for the other 2 unflagged comments. So, twice a day or even just once per day might be enough to avoid hitting the DB too hard?

I also chose to run at the minute 25, since other jobs run at the 0/12/20 minute on the hour, to try avoid putting too much pressure on the DB at once. Does this make sense?

Reply:
[Opinion] I definitely don't think this badge needs to be run hourly: it's not a big deal if a user doesn't receive their week streak badge **exactly** at the one week mark on the dot.

Reply:
Sounds great! Right now I'm feeling like running it once per day would be enough then. And we can always revisit this once we understand how it performs in the wild 🙂

Reply:
Agree on the idea that it only needs to run once a day.

is `"Awards 'Community Wellness' badges to users (runs at minute 25 past every 12th hour)"` visible to users or is it just the name of the cron job?  If the latter, I have no opinion.  If it shows up in the UI it's a little hard to parse.

Reply:
> is `"Awards 'Community Wellness' badges to users (runs at minute 25 past every 12th hour)"` visible to users or is it just the name of the cron job?

It's the latter: Only visible within the code and not visible in the web UI for end users in any way 👍 

It helps devs when looking at this file (after some time or when landing here for the first time) to quickly grasp how often each job runs, instead of having to parse the cron syntax in their (our) mind or having to use a tool for this 😄